cpp/src/core/sroptcryst.h

```diff
@@ -295,7 +279,7 @@ class srTOptCryst : public srTGenOptElem {
 **/
 	}
 
-	void FindDefOutFrameVect(double phEn, double hn, double ht, TVector3d& tv, TVector3d& sv, TVector3d& vZ1, TVector3d& vX1)
+	void FindDefOutFrameVect(double phEn, double hn, double ht, TVector3d& nv, TVector3d& tv, TVector3d& sv, TVector3d& vZ1, TVector3d& vX1) //ARF06062019 Introduce Vector3d& nv,
```

Comment:
Did not implement this (this function is called before propagation, so one can well accept having nv calculated in it as vector product: nv = tv x sv, instead of adding nv to arguments).